src/pytest_postgresql/janitor.py

```diff
@@ -92,7 +92,7 @@ def drop(self) -> None:
 
     @staticmethod
     def _dont_datallowconn(cur: cursor, dbname: str) -> None:
-        cur.execute("UPDATE pg_database SET datallowconn=false WHERE datname = %s;", (dbname,))
+        cur.execute(f'ALTER DATABASE "{dbname}" with allow_connections false;')
```

Comment:
I know these are only tests and so on, but we should rely on escaping psycopg2 does, just for the sake of keeping muscle memory.

Reply:
🤦🏻 yeah, that escaping and additional parenthesis....